### PR TITLE
Sample recipe spark

### PR DESCRIPTION
**Pull request by tumulurik**

### CODE REVIEW

File: recipes/scala/pom.xml [hunk 1]
```diff
@@ -27,6 +27,15 @@
...
 	</repositories>
+
+	<!-- Enable this for staged release of the artifact
+	<repositories>
+		<repository>
+			<id>staged-releases</id>
+			<url>https://oss.sonatype.org/content/repositories/comadobeplatform-1037</url>
+		</repository>
+	</repositories> -->
+
```
Comment (rjrussell77) [inline recipes/scala/pom.xml:38]: "staged release" - does this refer to a traditional "Stage" version?  1037 is dynamic so should it be here?  Can we shorten the `url` to  just `https://oss.sonatype.org/content/repositories`

Comment (tumulurik) [reply recipes/scala/pom.xml:38]: comadobeplatform-1037 is the released version of the authoring-sdk_2.11. 
Added it in pom just for reference for future iterations of releases and had it commented out. The url has to refer to this whole path as per the Sonatype's location of the artifact.

**Comment by rjrussell77** [comment]
Management is encouraging use of branch naming to indicate the ticket tracking the work (e.g. tumulurik:platml-1234).  Then, there can be easy cross-reference to the PR in the jira ticket.  (sometimes this is not ideal or possible but if the work is tracked in a jira ticket, this is the preferred workflow)  Check out https://jira.corp.adobe.com/browse/PLATML-3872 as a good example of this.

**Review by rjrussell77 — APPROVED** [review]
LGTM

File: recipes/scala/pom.xml (hunk 1 above)
Comment (rjrussell77) [reply recipes/scala/pom.xml:38]: Nice to have 'help' in the form of this comment so dev can be easier going forward.

**Comment by tumulurik** [comment]
https://jira.corp.adobe.com/browse/PLATML-3878

**Pull request merged**